contents/tutorials/next-steps-after-installing.md

```diff
@@ -0,0 +1,160 @@
+
+![Invite](../images/tutorials/next-steps-after-installing/invite.png)
+
+To make sure they don’t mess up all your hard work, you can change their access levels in your organization settings. Setting them to “member” will prevent them from deleting or changing the settings in your project or instances. You can also set up projects you don’t want others to access as private in the project settings. 
+
+When working with others, you likely will want to [filter them out](/tutorials/filter-internal-users) of the events. To do this, go to ”project settings”, scroll down to “filter out internal and test users”  and add filters such as host does not include local development addresses and email does not include your company’s domain.
+
+## Data where you want it
+
+Once everyone’s onboard with PostHog, you want the data to be where you all can see it. You want events and dashboards top of mind to help you improve your product. To do this, you can setup webhooks and subscriptions to your data and dashboards. 
+
+If there are key events or actions you want to track, such as a potential customer visiting your pricing page, you can have it trigger a webhook. This requires setting up the webhook in the service of your choice (like [Slack](/docs/integrate/webhooks/slack), [Teams](/docs/integrate/webhooks/microsoft-teams), [Zapier](/tutorials/how-to-connect-posthog-and-notion-with-zapier), or [Discord](/docs/integrate/webhooks/discord)). You can then add the webhook to your PostHog instance in “project settings” under “webhook integration.” Finally, go to data management, create or select an action, then enable “post to webhook when this action is triggered.”
+
+![Webhook](../images/tutorials/next-steps-after-installing/hook.png)
+
+You can also subscribe to dashboards in Slack or via email. Just go to your dashboard, click the three dots to the left of share, click “subscribe,” and “new subscription.” If you want to send dashboards to Slack, you’ll need to configure our [Slack integration](/docs/integrate/third-party/slack). If you want dashboards by email, just fill in the details like the name, email, and recurrence, then click “create subscription.” 
+
+Finally, if you want your data backed up or sent to an external service like [BigQuery](/apps/bigquery-export), [S3](/apps/s3-export), [Snowflake](/apps/snowflake-export), or many more, you can use one of our many [“data-out” apps](/apps). Just go to “browse apps,” search for the one you want, configure it, and start sending the data to the other places you want it.
+
+![Data-out apps](../images/tutorials/next-steps-after-installing/apps.png)
+
+## Experiment to improve your product
+
+Once you have some data, and you’ve had a chance to gain some insights from it, it’s time to start using those insights to improve your product. A great way to do this is with experiments.
+
+Experiments is a feature that helps you test if changes improve your product in the ways you care about.
```

Comment:
```suggestion
Experiments in PostHog help you test if changes improve your product in the ways you care about.
```

Ha, this tense is tricky. It should be 'Experiments are...', but Experiments is actually the name of the feature so 'Experiments is...' is also correct. IMHO, just avoid this phrasing.